.gitignore

```diff
@@ -30,4 +30,4 @@ coverage.txt
 kustomize/network/etcd
 apiserver.local.config
 tmp/
-
+kubesphere.yaml
```

Comment:
Where this file comes from? Do any programs generate it?

Reply:
It is a configure file for the developer.  Can we provide a template file? @yuswift

Reply:
Which folder do you suggest we put it in?

Reply:
add `kubesphere.yaml.tpl` file to this repo, so the developer can just rename it and configure vars to run the project.

```yaml
cat kubesphere.yaml.tpl 
kubernetes:
  kubeconfig: "/root/.kube/config"
  master: https://192.168.0.5:6443
  $qps: 1e+06
  burst: 1000000
authentication:
  authenticateRateLimiterMaxTries: 10
  authenticateRateLimiterDuration: 10m0s
  loginHistoryRetentionPeriod: 168h
  maximumClockSkew: 10s
  multipleLogin: False
  kubectlImage: kubesphere/kubectl:v1.0.0
  jwtSecret: "xxx"
...
```

https://github.com/kubesphere/community/blob/master/developer-guide/development/quickstart.md#understand-kubesphere-cores-configuration

Reply:
Good suggestion.